src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -249,7 +250,11 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
         }
       })
       .catch(error => {
-        MessageCenter.add(API.getErrorMsg('Could not fetch Service Details', error));
+        MessageCenter.add(
+          API.getInfoMsg('Could not fetch 3scale info. Turning off 3scale integration.', error),
```

Comment:
is it accurate to say that the integration is switched off in this scenario?

Reply:
From the perspective of the user, yes.
He/she won't have access to 3scale integration through Kiali as he/she doesn't have right access.

Reply:
It might give the impression that the integration will be disabled, which is not the case here. I think it's better to say something that describes the situation more accurately - user don't have the right permissions.

Reply:
We are following the same message used where Jaeger and Grafana integration for consistency.
As it's similar scenario.

Reply:
so perhaps the messages there need to be changed a bit as well :) 
@jotak @aljesusg